packages/account/tests/account.spec.ts

```diff
@@ -40,6 +42,78 @@ describe("Account Tests", () => {
     expect(address).toBeTruthy();
   });
 
+  it("should pickup the rpcUrl from viem wallet and ethers", async () => {
+    const {
+      chainId,
+      bundlerUrl,
+      viemChain,
+      whale: { privateKey, viemWallet: originalViemSigner, ethersSigner: originalEthersSigner },
+    } = ganache;
+
+    const newRpcUrl = "http://localhost:8545";
+    const defaultRpcUrl = viemChain.rpcUrls.public.http[0]; //http://127.0.0.1:8545"
+
+    const ethersProvider = new JsonRpcProvider(newRpcUrl);
+    const ethersSignerWithNewRpcUrl = new Wallet(privateKey, ethersProvider);
+
+    const accountOne = privateKeyToAccount(privateKey);
+    const walletClientWithNewRpcUrl = createWalletClient({
```

Comment:
what is the difference between viem's createPublicClient and createWalletClient?

Reply:
They just do different things. They both come from viem. In ethers terms it's almost the same difference as a provider and a signer.